### PR TITLE
Bdex/57603: follow up - added more evss provider tests and local file requirement

### PR DESCRIPTION
**Pull request by ahay-agile6**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary

1. [add more evss provider unit tests](https://github.com/department-of-veterans-affairs/vets-api/commit/0c4fbf3b91267a7b5e131f9b9a31b2ff13d4a757) 
-- for fun
2. [require dto file for rails console commands to work](https://github.com/department-of-veterans-affairs/vets-api/commit/59c138145a419e612c3b052714e61bca20a45d3a) 
-- this isn't failing in the tests nor in ArgoCD.
-- only affects `rails c` commands in local environment

## Related issue(s)
- department-of-veterans-affairs/va.gov-team#57603
- [*Link to ticket created in va.gov-team repo*](https://app.zenhub.com/workspaces/disability-experience-63dbdb0a401c4400119d3a44/issues/gh/department-of-veterans-affairs/va.gov-team/57603)

## Testing done

- added unit tests only
- added file requirement. to test that is is working, use a `rails console` and attempt to use  `LighthouseRatedDisabilitiesProvider#get_rated_disabilities`. this was failing before, now it's not

## Screenshots
_Note: Optional_

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: spec/lib/disability_compensation/providers/rated_disabilities/evss_rated_disabilities_provider_spec.rb [hunk 1]
```diff
@@ -1,11 +1,45 @@
...
+    EVSS::AuthHeaders.new(current_user).to_h
+  end
+
+  before do
+    Flipper.disable(ApiProviderFactory::FEATURE_TOGGLE_RATED_DISABILITIES)
+    allow_any_instance_of(Auth::ClientCredentials::Service).to receive(:get_token).and_return('blahblech')
+  end
+
+  after do
+    Flipper.disable(ApiProviderFactory::FEATURE_TOGGLE_RATED_DISABILITIES)
```
Comment (navaeddie) [inline spec/lib/disability_compensation/providers/rated_disabilities/evss_rated_disabilities_provider_spec.rb:23]: seems a little odd we're disabling the same feature flag before and after this runs. Just flagging to make sure that was intentional.

Comment (ahay-agile6) [reply spec/lib/disability_compensation/providers/rated_disabilities/evss_rated_disabilities_provider_spec.rb:23]: this is intentional.

the way vets-api has Flipper set up is to disable by default in all environments _**except for**_ `test`.

we need to explicitly set these to test the appropriate and expected behavior :smile:

Comment (ahay-agile6) [reply spec/lib/disability_compensation/providers/rated_disabilities/evss_rated_disabilities_provider_spec.rb:23]: also, the tests do not reset the feature flags, so, locally, on subsequent runs, if there is a feature flag disabled, the database does not set it back to enabled and vice versa.

the only option is to ensure that it is set the way we need it in the test.

**Review by sethdarragile6 — APPROVED** [review]
lgtm!

**Review by navaeddie — APPROVED** [review]

**Review by ryan-mcneil — APPROVED** [review]
👌

**Pull request merged**